frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -455,13 +456,14 @@ class BitBox02 extends Component<Props, State> {
             key="connection"
             fullscreen
             textCenter
+            verticallyCentered
             withBottomBar
-            width="600px">
+            width="690px">
             <ViewHeader title={t('button.unlock')}>
               <p>{t('bitbox02Wizard.stepConnected.unlock')}</p>
             </ViewHeader>
             <ViewContent fullWidth>
-              {attestationResult === false ? (
+              {attestationResult === true ? (
```

Comment:
Was this put into the commit by mistake? Looks like a regression to me cc @shonsirsha @thisconnect

Reply:
Yes by looking at the code this isn't right. Or at least this PR shouldn't concern anything about `attestationResult`. I'll make a PR to revert this bit. good catch!

Reply:
Possibly it was added when [this](https://github.com/shonsirsha/bitbox-wallet-app/pull/1) was created and this attestationResult part was only meant for testing during development. Feel free to clarify this though @thisconnect :)